src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -224,20 +231,16 @@ def get_tags(
 
 def get_context(group: Group) -> str:
     context_text = ""
-    context = {
-        "Events": get_group_global_count(group),
-        "Users Affected": group.count_users_seen(),
-        "State": SUBSTATUS_TO_STR.get(group.substatus, "").replace("_", " ").title(),
-        "First Seen": time_since(group.first_seen),
-    }
-    if group.issue_type in REGRESSION_PERFORMANCE_ISSUE_TYPES:
-        # another short term solution for non-error issues notification content
-        return context_text
-
-    if group.issue_category in [GroupCategory.ERROR, GroupCategory.PERFORMANCE]:
-        for k, v in context.items():
-            if k and v:
-                context_text += f"{k}: *{v}*   "
+    context = group.issue_type.notification_config.context
+
+    context_dict = {}
+    for c in context:
+        if c in SUPPORTED_CONTEXT_DATA:
+            context_dict[c] = SUPPORTED_CONTEXT_DATA[c](group)
+
+    for k, v in context_dict.items():
+        if k and v:
```

Comment:
The key `k` will always exist though right, or are there situations where it's empty string / 0? We're mainly worried about if the value is falsy?

Reply:
yes i believe we are worried if the value is falsey. i could do `if v` too